pkg/synopsysctl/cmd_create.go

```diff
@@ -91,31 +91,34 @@ var createAlertCmd = &cobra.Command{
 		alertName := args[0]
 		helmReleaseName := fmt.Sprintf("%s%s", alertName, globals.AlertPostSuffix)
 
-		ok, err := util.IsNotDefaultVersionGreaterThanOrEqualTo(cmd.Flag("version").Value.String(), 5, 3, 1)
-		if err != nil {
-			return err
-		}
-		if !ok {
-			return fmt.Errorf("creation of Alert instance is only suported for version 5.3.1 and above")
-		}
-
 		// Get the flags to set Helm values
 		helmValuesMap, err := createAlertCobraHelper.GenerateHelmFlagsFromCobraFlags(cmd.Flags())
 		if err != nil {
 			return err
 		}
 
 		// Update the Helm Chart Location
-		newChartVersion := "" // pass empty to SetHelmChartLocation if the default version should be used
+		newChartVersion := "" // pass empty to UpdateHelmChartLocation if the default version should be used
 		if cmd.Flags().Lookup("version").Changed {
 			globals.AlertVersion = cmd.Flags().Lookup("version").Value.String()
 			newChartVersion = globals.AlertVersion
 		}
-		err = SetHelmChartLocation(cmd.Flags(), globals.AlertChartName, newChartVersion, &globals.AlertChartRepository)
+		err = UpdateHelmChartLocation(cmd.Flags(), globals.AlertChartName, newChartVersion, &globals.AlertChartRepository)
 		if err != nil {
 			return fmt.Errorf("failed to set the app resources location due to %+v", err)
 		}
 
+		ok, err := util.IsNotDefaultVersionGreaterThanOrEqualTo(globals.AlertVersion, 5, 3, 1)
```

Comment:
passing globals.AlertVersion will always pass right. Did we need to use version variable?